File: BOTAIML.VisionBot.Monitoring.Web/ViewModels/LiveStreamViewModel.cs
using BOTAIML.VisionBot.Monitoring.Web.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BOTAIML.VisionBot.Monitoring.Web.ViewModels
{
    public class LiveStreamViewModel
    {
        public int Id { get; set; }

        public string CameraCode { get; set; }

        public bool IsEnabled { get; set; }

        public CameraLiveStatus CameraLiveStatus { get; set; }

        public CameraComponentStatus CameraStatus { get; set; }

    }
}

File: BOTAIML.VisionBot.Monitoring.Web/ClientApp/src/app/api/models/live-stream-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { CameraComponentStatus } from './camera-component-status';
import { CameraLiveStatus } from './camera-live-status';
export interface LiveStreamViewModel {
  cameraCode?: null | string;
  cameraLiveStatus?: CameraLiveStatus;
  cameraStatus?: CameraComponentStatus;
  id?: number;
  isEnabled?: boolean;
}
